dockerfiles/frameworks/src/flow/30.tz.ini

```diff
@@ -0,0 +1 @@
+date.timezone = "Europe/Warsaw"
```

Comment:
[Minor] Should we go with something more neutral, such as `UTC`?

Reply:
Actually the default TZ was UTC - but flow framework complained :/ 
Picked one at random ;)
Any TZ suggestion other than UTC ? :)

Reply:
Per your suggestion :) I'll address this in laravel PR